### DISCUSSION TITLE
The prefix is already registered

### DISCUSSION THREAD

**Discussion by sholman** [datalust/seq-tickets#1574] [Help] [answered]
I am trying to set a custom ingestion port and a custom domain.
I need to be able to log messages from the local server using localhost:45341 and from externally using a FQDN. 

I'm using the following

````
seq bind-ssl --thumbprint="ssl thumbprint"
seq bind-ssl --port=45341 --thumbprint="ssl thumbprint"
seq config set -k api.listenUris -v https://seq.mydomain.com.au,https://seq.mydomain.com.au:45341,http://localhost:45341
seq service restart
````

The restart fails with a ""Error running the server application","@l":"Fatal","@x":"Microsoft.AspNetCore.Server.HttpSys.HttpSysException (183): The prefix 'https://seq.mydomain.com.au:45341/' is already registered."

**Comment by nblumhardt** [comment, marked as answer]
Thanks for the note! This happens when switching hostnames/ports after the initial install, and where the service account doesn't itself have enough privileges to add the HTTP namespace prefix reservations.

You can add these yourself via `netsh`, using:

```
netsh http delete urlacl url=https://seq.mydomain.com.au:45341/
netsh http add urlacl url=https://seq.mydomain.com.au:45341/ user=SeqDefaultInstance
```

(Assuming that's the correct username for the Seq service - may require some light tinkering :-))

Let me know if this helps,
Nick

**Reply by sholman** [reply]
Thanks Nick. Will give it a go
